### PR TITLE
Fix bigQuerySelect deprecation message

### PR DESCRIPTION
**Pull request by ClaireMcGinty**

### CODE REVIEW

File: scio-bigquery/src/main/scala/com/spotify/scio/bigquery/syntax/ScioContextSyntax.scala [hunk 1]
```diff
@@ -53,7 +53,7 @@ final class ScioContextOps(private val self: ScioContext) extends AnyVal {
    * behavior, start the query string with `#legacysql` or `#standardsql`.
    */
   @deprecated(
-    "this method will be removed; use bigQuery(Query(sql), flattenResults) instead",
+    "this method will be removed; use bigQuerySelect(Query(sql), flattenResults) instead",
```
Comment (ClaireMcGinty) [inline scio-bigquery/src/main/scala/com/spotify/scio/bigquery/syntax/ScioContextSyntax.scala]: on a related note, the new `bigQuerySelect` method doesn't have a default value for `flattenResults` like this method does -- I'm in favor of adding it, thoughts?

Comment (regadas) [reply scio-bigquery/src/main/scala/com/spotify/scio/bigquery/syntax/ScioContextSyntax.scala]: yeah oops; adding default param it's not enough though we will get method overload issue. 

maybe we can add an extra method?
```scala
  def bigQuerySelect(sqlQuery: Query): SCollection[TableRow] = ???
```

Comment (ClaireMcGinty) [reply scio-bigquery/src/main/scala/com/spotify/scio/bigquery/syntax/ScioContextSyntax.scala]: ahhh right we can do an extra method! I'll add it into this PR in a minute 👍

Comment (regadas) [reply scio-bigquery/src/main/scala/com/spotify/scio/bigquery/syntax/ScioContextSyntax.scala]: cool cool

Comment (regadas) [reply scio-bigquery/src/main/scala/com/spotify/scio/bigquery/syntax/ScioContextSyntax.scala]: oh sorry LOL you said another PR

Comment (ClaireMcGinty) [reply scio-bigquery/src/main/scala/com/spotify/scio/bigquery/syntax/ScioContextSyntax.scala]: no worries :)

**Pull request merged**